c_gen/c_code_gen.py

```diff
@@ -32,7 +32,7 @@
 
 import sys
 import c_gen.of_g_legacy as of_g
-import c_match
+from . import c_match
```

Comment:
I'm not 100% sure about the relative imports, I feel like I've been told off before for using them :) 

Maybe @ronaldchl or @JanStrauss have some recommendations on their use/alternatives

Reply:
yes, let's have some python3 experts weigh in.
i did most of the conversions because they were flagged by 2to3.
i think this was a sibling import that was suggested by 2to3 to be converted to relative, but other than that, i have no great opinion.

Reply:
I prefer having one root package and using absolute imports but relative imports are ok if the project isn't too large. Additional nice-to-have: grouping the imports according to the pep 8 guidelines https://www.python.org/dev/peps/pep-0008/#imports

Reply:
Jan, thx for your comment and pointer to PEP8.  since these are mostly sibling imports, i'll leave them as they are now and take a better look at PEP8 to see what else is applicable

Reply:
actually, this wasn't so bad; i replaced a lot of "from . import" statements with "from XXX import", but i left the other "from . import" statements in the generated code.